src/extension/extensionWeb.ts

```diff
@@ -50,6 +50,17 @@ export async function activate(context: vscode.ExtensionContext) {
     backendProxy: backendProxy,
   });
 
+  vscode.commands.registerCommand('kaoto.open.textualeditor', async() => {
+    if (kieEditorStore.activeEditor !== undefined) {
+      const doc = await vscode.workspace.openTextDocument(kieEditorStore.activeEditor?.document.document.uri);
```

Comment:
it is really ".document.document" syntax? it looks a bit strange, so I am just asking..

Reply:
yes, I confirm it is really this API